### ISSUE TITLE
Flagging guides to **lets-plot**

### ISSUE THREAD

**Issue by aeturrell** [JetBrains/lets-plot#860]
I'm not sure if you have a community links page (?) but I wanted to flag two resources I've put together on **lets-plot** and how to use it:

- Throughout in [Python for Data Science](https://aeturrell.github.io/python4DS), a port of R for Data Science to Python.
- [Coding for Economists](https://aeturrell.github.io/coding-for-economists/vis-letsplot.html), in which one of the data visualisation pages uses **lets-plot**.

**Comment by alshan** [comment]
Wow! Thanks for sharing!
We absolutely will add links at lets-plot.org.

**Comment by alshan** [comment]
Hi, @aeturrell 
I would like to let you know we are now using your books as the [de-facto user guide](https://lets-plot.org/python/index.html#python-index-user-guide) for lets-plot :) If you don't mind of course.

BTW I just finished your "The Star Builders" book which is 5-star work! Sad to see however that as the time pass most optimistic promises of viable prototipes (or even commercialisation) of  nuclear fusion power generators didn't come true.
The net energy gain was indeed achieved but that is if the energy loss in the lasers is taken out of the equation - doesn't fit the bill either :(

**Comment by aeturrell** [comment]
Oh wow, this is absolutely amazing news, thank you! I don't mind at all, I'm delighted that the guides can be useful to people. 

And that's a very generous review of The Star Builders, thank you, I'm super pleased you enjoyed it! If you have a minute to leave a Goodreads or Amazon review too, I'd be super grateful—it genuinely helps other people who might enjoy the book find it. Yes, it's a shame about the over-promising of the private sector fusion outfits. I still believe they have a lot to contribute but I hope they'll be a bit more strategic about their promises (they should be, if they're in it for the long run). The laser facility at NIF was never designed to get net energy gain including the lasers too (in fact Congress told them not to do this, but to demonstrate it just with the target first). One would have built it with very different lasers if that were the aim. So I still count the NIF breakthrough as a huge win and step forward! What matters now is whether we, as a society, follow up on it. Exciting times!

**Comment by alshan** [comment]
Absolutely, I'm keeping track of my books on Goodreads, but apart from rating them, this is my first book review ever :)
https://www.goodreads.com/review/show/6258370366

**Comment by aeturrell** [comment]
Amazing, this is so kind and generous—thank you!

**Comment by alshan** [comment]
Thanx for the books :)
